### DISCUSSION TITLE
Usage of Github Projects

### DISCUSSION THREAD

**Discussion by dgduncan** [mochi-mqtt/server#304] [General]
I have been noticing some backlogged items coming out of some PRs. What is everyone's opinions on the usage of Github projects to bucket some of these things to better track? For example, perhaps a project for `File Based Configuration` and so on. That way some of these things such as the recent request to change some error logging can be tracked and put into a backlog.

An example of one I somewhat like is the zig projects page - [here](https://github.com/ziglang/zig/projects?type=classic)

**Comment by mochi-co** [comment]
This would be ideal. I had intended to try this but have been somewhat sidetracked, so I fully support the addition of projects 👍🏻

**Reply by dgduncan** [reply]
Sounds good! I can create some basic ones as experiments and we can iterate from there. I do not have access to create projects in this repository or organization so feel free to assign me with some project permissions when you have the time or if you do not wish to do that create them. Either works for me :)

**Reply by mochi-co** [reply]
I had a look quickly and couldn't figure out how manage permissions for this without upgrading the github organisation. I've created a new project `File Based Configuration` and added you as admin, so you should have full permissions to configure it as you wish and invite collaborators :) 

https://github.com/orgs/mochi-mqtt/projects/2

This will be a good case study for other projects.

**Comment by werbenhu** [comment]
@dgduncan The task lists and task cards on the Project Boards look really good :heart:.

**Comment by dgduncan** [comment]
@werbenhu @mochi-co I just started experimenting with projects and milestones. Please bare with me while I mess around with things and feel free to ask any question or throw around suggestions!
